EMBRACE_Client/ConditionSetup.h

```diff
@@ -0,0 +1,39 @@
+//
+//  ConditionSetup.h
+//  EMBRACE
+//
+//  Created by Jonatan Lemos Zuluaga (Student) on 10/24/14.
+//  Copyright (c) 2014 Andreea Danielescu. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+////Describes the condition in which the app will be deployed
+//typedef enum Condition {
+//    MENU,
+//    HOTSPOT,
+//    CONTROL,
+//    OTHER,
+//} Condition;
+//
+////Defines the types of language conditions to be used
+//typedef enum Language {
+//    ENGLISH,
+//    BILINGUAL
+//} Language;
+
+//NSString* condition = @"Control";
+
+@interface ConditionSetup : NSObject {
+
+    NSString *condition;
```

Comment:
Add comments to explain what condition and language are used for. Include what values they accept--or better, use the typedef enums that were commented out above. (Change Condition to be either EMBRACE or CONTROL, while Language can stay the same.)